### ISSUE TITLE
Optimize product bulk mutations DB queries

### ISSUE THREAD

**Issue by SzymJ** [saleor/saleor#13035]
Currently, Saleor API allows assigning attributes and values to products/variants in bulk mutations by using `id`, `externalReference` and `value`. Additionally, we allow to create a new, not existing values in those mutations. Unfortunately, the part related to `attributes/values` in those mutations is the most complex and very difficult to optimize and maintain. We think that in product/variant bulk mutations we should allow only use of already created attribute values. If a value does not exist we should throw an error or skip the value and save the product without this value depending on `ERROR_POLICY`. That will simplify the code and boost the performance of bulk product mutations. Then we should add `AttributeBulkCreate` and `AttributeBulkUpdate` mutations that allow users to create and manage all attributes that they need in the product/variants creation process. Those two mutations not only help optimize product bulk mutations but will be very helpful for clients in importing attribute sets they have already defined in other systems.

To sum up:
- we shouldn't allow creating new attributes values in product/variants bulk mutation 
- we should add `AttributeBulkCreate/Update` mutations to easily import and maintain attributes and values(this will be very useful feature for Saleor users) 
- changes above allow for better optimization of product/variants bulk mutations and easier code maintenance

@timuric @fowczarek @maarcingebala

**Comment by maarcingebala** [comment]
Tickets to track new work on new mutations:
- https://github.com/saleor/saleor/issues/13375
- https://github.com/saleor/saleor/issues/13374
